third_party/nvfuser/csrc/grouped_reduction.cpp

```diff
@@ -35,9 +46,10 @@ bool hasMatchingTransformations(TensorView* ref, TensorView* other) {
 }
 
 // Validate grouping of reductions and return a new max producer position
-void validateReductionGrouping(
+bool validateReductionGrouping(
```

Comment:
Changed the validation to not fail unless instructed. Intended to be used by schedulers.